glass/_array_api_utils.py

```diff
@@ -31,15 +36,68 @@ def get_namespace(*arrays: NDArray[Any] | Array) -> ModuleType:
     return namespace
 
 
-UnifiedGenerator: TypeAlias = np.random.Generator | Generator
-
-
-def rng_dispatcher(array: NDArray[Any] | Array) -> UnifiedGenerator:
+def rng_dispatcher(array: NDArray[Any] | JAXArray) -> UnifiedGenerator:
     """Dispatch RNG on the basis of the provided array."""
     backend = array.__array_namespace__().__name__
     if backend == "jax.numpy":
-        return Generator(seed=42)
-    if backend in {"numpy", "array_api_strict"}:
+        return glass.jax.Generator(seed=42)
+    if backend in "numpy":
```

Comment:
Feel like this might have been copied over unintentionally?
```suggestion
    if backend == "numpy":
```